pecorino.gemspec

```diff
@@ -30,10 +30,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # Uncomment to register a new dependency of your gem
-  spec.add_dependency "activerecord", "~> 7"
+  spec.add_dependency "activerecord", "~> 8"
```

Comment:
Let's make both of these development dependencies instead of runtime dependencies. This gem should work fine even with Rails 6 but you would need to set up Appraisal to run the tests on GH actions for this. If we make AR ~> 8 a hard dependency people won't be able to use Pecorino with Rails 7 anymore.

Alternatively, we can set `>= 7`, `< 9` as constraints?